Release v0.4.7

igr-santos opened the pull request.

igr-santos committed:
Add reusable button. Issue #497

igr-santos committed:
Add reusable float div. Issue #497

igr-santos committed:
Add reusable settings form. Issue #497

igr-santos committed:
Use reusable components in pages of settings. Issue #497

igr-santos committed:
Use reusable components in pages of settings. Issue #497

igr-santos committed:
Change strategy to launch mobilization for use reusable components. Issue #497

gabrielrtakeda committed:
Move EditorSlate component styles to a separated file #511

gabrielrtakeda committed:
Move EditorSlate component styles to a separated file #511

gabrielrtakeda committed:
Move EditorSlate component styles to a separated file #511

gabrielrtakeda committed:
Move EditorSlate component styles to a separated file #511

gabrielrtakeda committed:
Remove unused EditorSlate component classNames and styles #511

gabrielrtakeda committed:
Remove unused EditorSlate component classNames and styles #511

gabrielrtakeda committed:
Remove unused EditorSlate component classNames and styles #511

gabrielrtakeda committed:
Fix EditorSlate component toolbar z-index; Close #511

gabrielrtakeda committed:
Fix EditorSlate component toolbar z-index; Close #511

gabrielrtakeda committed:
Fix EditorSlate component toolbar z-index; Close #511

gabrielrtakeda committed:
Add standard-loader to webpack config #508

gabrielrtakeda committed:
Fix some files with standard js #508

gabrielrtakeda committed:
Upgrade `babel-eslint` package to v7.2.1; Make standard undestands some global variables #508

gabrielrtakeda committed:
Fix standardjs warnings; Close #508

gabrielrtakeda committed:
Fix tests after fix standardjs warnings; Close #508

igr-santos committed:
Remove any files nonused. Issue #506

igr-santos committed:
Remove any files nonused. Issue #506

gabrielrtakeda committed:
Fix failing tests #511

gabrielrtakeda committed:
Redefine community paths methods #489

gabrielrtakeda committed:
Add community settings pages links into sidebar when does not editing a mobilization; Close #489

gabrielrtakeda committed:
Add community settings pages links into sidebar when does not editing a mobilization; Close #489

gabrielrtakeda committed:
Fix stateless component coding style; Add tests to sidebar component; Close #489

gabrielrtakeda committed:
Remove community settings link located on top of sidebar component; Close #489

gabrielrtakeda committed:
Add textarea to customize the message of whatsapp share #487

gabrielrtakeda committed:
Merge branch 'hotfix-v0.4.6' into develop

gabrielrtakeda committed:
Use previously configured whatsapp text into form widget component #487

gabrielrtakeda committed:
Pass widget object through donation widget component #487

gabrielrtakeda committed:
Pass widget object through donation widget component; Close #487

gabrielrtakeda committed:
Install `reapop` and its theme package to display notifications #500

gabrielrtakeda committed:
Fix pressure widget city input placeholder typo; Close #519

igr-santos committed:
Move navbar component to mobrender. Issue #506

igr-santos committed:
Remove actions and reducers from mobilizations module. Issue #506

igr-santos committed:
Remove blocks unused files from mobilizations module. Issue #506

igr-santos committed:
Remove blocks unused files from mobilizations module. Issue #506

gabrielrtakeda committed:
Combine reapop notification reducers with our existing reducers #500

gabrielrtakeda committed:
Apply reapop notification system component into custom domain page #500

gabrielrtakeda committed:
Add a util file that will contain all notification message type #500

igr-santos committed:
Remove widgets unused files from mobilizations module. Closes #506

igr-santos committed:
Remove draft unused files from mobilzitions module. Closes #506

gabrielrtakeda committed:
Fix whatsapp share button box-sizing; Close #513

gabrielrtakeda committed:
Revert "Remove community settings link located on top of sidebar component; Close #489"

This reverts commit 8efb025435e6d073dfea96e9c6e82a9bbd43bee9.

gabrielrtakeda committed:
Show error notification when request to api on donation fails; Close #500

gabrielrtakeda committed:
Fix standardjs linting warning #500

lpirola committed:
Add simple docker support

lpirola committed:
Setup config to drone

lpirola committed:
Fix when ci plugin is fired

lpirola committed:
Check if s3 is configured before allow webpack upload

lpirola committed:
Set missing env vars

igr-santos committed:
Move components reusable to ux module. Issue #497

gabrielrtakeda committed:
Fix React invariant error when use Draft.JS editor #524

gabrielrtakeda committed:
Make widget finish custom message background and text color same on both editor and public view; Close #524

gabrielrtakeda committed:
Update `slate-editor` package version number to v2.2.8, to allow multiple uploads of the same image; Close #523

igr-santos committed:
Fix tell a friend when widget settings null

igr-santos committed:
Change forms and buttons in widget settings to reusable component. Issue #497

igr-santos committed:
Fix redirect to launch mobilization. Closes #497

igr-santos committed:
Bumped version number to 0.4.7

igr-santos committed:
Make step components and test on /playground-js page

gabrielrtakeda committed:
Fix AWS S3 bucket name when import main.css file, using environment variable

gabrielrtakeda commented:
It needs to fix the widget custom finish message slate-editor colorpicker z-index:

<img width="1154" alt="screen shot 2017-04-03 at 4 35 18 pm" src="https://cloud.githubusercontent.com/assets/5435389/24627531/8ec02cde-188b-11e7-9e67-da66395a8964.png">

gabrielrtakeda committed:
Fix content widget slate-editor toolbar z-index

igr-santos committed:
Apply a simple style to header

igr-santos committed:
Add README for step components

gabrielrtakeda committed:
Group PageCentralizedStepsLayout component into a folder #531

gabrielrtakeda committed:
Add new FormRedux layout component called `FlatForm` #531

gabrielrtakeda committed:
Add launch mobilization page #531

gabrielrtakeda committed:
Move page centralized layout component into layout folder #531
- Split page centralized layout title component;
- Fix some layout issues in launch mobilization page

gabrielrtakeda committed:
Refactoring mobilization's domain configuration form into a reusable component #531

gabrielrtakeda committed:
Make launch mobilization button redirects to launch flow route #531

gabrielrtakeda committed:
Add sidebar item that redirects to mobilization list page; Close #533

manuellaos committed:
Now videos on mobile are responsive close #512

lpirola committed:
Add more time to ssh timeout

gabrielrtakeda committed:
Add StepsContainerStack component to render steps in order, instead of render all steps at the same time #531

gabrielrtakeda committed:
Make launch mobilization page works with steps components stack; Inject step validation #531

gabrielrtakeda committed:
Make StepsContainerStack component accepts validate function injection #531

gabrielrtakeda committed:
Add launch mobilization domain page to enhance the speed of development #531

gabrielrtakeda committed:
Make StepsContainerStack component display only current step #531
- StepContent component now accepts style prop

gabrielrtakeda committed:
Make FormDomain component accepts injection of custom validation that merges with default validation #531
- Implements FormDomain components both on launch mobilization form and on mobilization settings
- Migrate ReduxForm assembly from page.connected files to page components

gabrielrtakeda committed:
Find chrome tab URL by regex match; Close #536

gabrielrtakeda committed:
Add second step of launch mobilization flow #531
- Disacoplated share form, generating a new component called FormShare

gabrielrtakeda committed:
Change strategy to jump to next step using steps validation #531

gabrielrtakeda committed:
Change strategy how jump to the next step using validations

gabrielrtakeda committed:
Adjust launch mobilization steps forms layout #531

gabrielrtakeda committed:
Adjust the reusable FormShare component in mobilization settings page #531

gabrielrtakeda committed:
Create page that ends the launch mobilization flow #531
- Add next function that redirects to last step of the launch mobilization flow after share form submit
- Move PageDomain component implementation to launch mobilization route page
- Increase the server build timeout to 2'

gabrielrtakeda committed:
Fix the StepsContainerStack component steps state refresh

gabrielrtakeda committed:
Show async request as `saving` in buttons of the launch mobilization flow #531

gabrielrtakeda committed:
Rewrite the rule to display the launch/launched mobilization button on sideba #531

gabrielrtakeda committed:
Create rule to hide the domain tab on mobilization settings when it is unlaunched; Close #531

gabrielrtakeda committed:
Fix tests after implementation of launch mobilization flow; Close #531

gabrielrtakeda committed:
Fix tests after implementation of StepsContainerStack component

gabrielrtakeda committed:
Update StepsContainerStack component docs

gabrielrtakeda committed:
Fix the way that the main.css is linked into server index file

gabrielrtakeda committed:
Fix error message when login auth fails; Enhance the way that retry error message appears; Close #372

gabrielrtakeda committed:
Add twitter cards markup tags into mobilization's public view; Close #532

igr-santos committed:
Add container to settings community page

lpirola committed:
Add better dev dockerfile env

Change container name to deploy

lpirola committed:
Fix tag name

lpirola committed:
Change dokku tag to deploy

lpirola committed:
Remove circleci config and set production deploy

lpirola committed:
Add var to config in build time

lpirola committed:
Add required missing env vars in build time

lpirola committed:
Rename api_url app name

gabrielrtakeda committed:
Remove console.log
